### DISCUSSION TITLE
Volume and brightness controls should increment by 10 instead of 1

### DISCUSSION THREAD

**Discussion by jaredreich** [351ELEC/351ELEC#796] [Ideas]
Loving 351ELEC on my RG351V, one issue I've noticed is when pressing volume up/down and FN + volume up/down (for brightness) the incremented amounts are tiny, like 1/100. So to change volume/brightness, one has to push the buttons like 50 times or hold them for many seconds. How about the increment being 10/100 instead of 1/100? Or maybe even a setting in the options like "Volume Increment Amount: 10" (range of 1-25 or something)

**Comment by MrScottyTay** [comment]
I personally would hate it if it was 10 only since I usually set my brightness to the lowest possible (would actually love it if it's lowest was even lower) so that would do my head in if it would jump from 10 to 0 with no variance in between. However this is still a great idea in terms of having a setting to change said variance.

**Comment by whizbo** [comment]
Maybe an "acceleration" on the increment would allow for coarse and fine adjustment in a timely manner.  Press the button and for the first 1 second it does a single point increment, hold the button longer and it goes to a 5 point increment, longer still 10 point,..... etc.  Kinda like when you delete blocks of text on your phone.  This is all very easy for me to explain, but I assume much harder to code, so please take my ignorance with a grain of salt.  Thanks

EDIT:  The website says "As you hold longer, the speed of brightness change increases."   I have not tested this, but I may have spoken too soon.
